application/comit_node/src/settings/mod.rs

```diff
@@ -1,9 +1,9 @@
 mod serde;
 
 use crate::seed::Seed;
+use ::serde::Deserialize;
```

Comment:
❌ can we avoid relative path please

Reply:
I think that just indicates that it's an external crate. Otherwise the name clashes with the local module `serde`. We have this same situation in several places.

Reply:
Addressed :)